repos/system_upgrade/el7toel8/actors/removeresumeservice/tests/test_removeresumeservice.py

```diff
@@ -8,8 +8,8 @@
 
 
 @pytest.mark.skipif(
-    distro.linux_distribution()[0] == 'Fedora',
-    reason='default.target.wants does not exists on Fedora distro',
+    distro.linux_distribution()[0] not in ('CentOS Linux', 'Red Hat Enterprise Linux Server'),
+    reason='default.target.wants does not exists.',
```

Comment:
this is not connected to rpm distributions.

Reply:
Solving it in a different manner, where we are testing whether the default.targets.wants really exists, the test running under the root and at the end, it is marked as a destructive test